### ISSUE TITLE
Backend loadbalancer documentation incorrect for name field

### ISSUE THREAD

**Issue by jaoracle** [oracle/terraform-provider-baremetal#139]
## Description of the bug:
terraform: v0.9.4
provider: commit 2ba4c0902bd8cb753419fd0b6620adf8630f49a9

The name field in [baremetal_load_balancer_backend](https://github.com/oracle/terraform-provider-baremetal/blob/master/docs/resources/loadbalancer/backend.md) only works as a computed value and does not accept a user value. Documentation should either remove that field or mark it as computed.

If a user value is provided it gives the following error:

```
1 error(s) occurred:

* baremetal_load_balancer_backend.etcd_lb: "name": this field cannot be set
1 error(s) occurred:

* baremetal_load_balancer_backend.etcd_lb: "name": this field cannot be set
```

## A self-contained terraform file that reproduces the issue:

(If this is not provided, your ticket will be closed)

[tflb.zip](https://github.com/oracle/terraform-provider-baremetal/files/1050803/tflb.zip)

**Comment by codycushing** [comment]
Thanks, doc updated with PR #157

**Comment by codycushing** [comment]
Fixed in v1.0.14
